api/incidents/incidentsRouter.js

```diff
@@ -291,4 +292,86 @@ router.post('/fetchfromds', async (req, res) => {
   }
 });
 
+router.get('/download', async (req, res) => {
+  try {
+    // Get Incidents from Database:
+    const incidents = await Incidents.getAllIncidents();
+    // Create Field Names:
+    const fields = [
```

Comment:
This file is a bit long already. Let's try taking this array and putting it into a resource/utility type file and them importing it here instead.